avocado/utils/ssh.py

```diff
@@ -248,3 +249,20 @@ def copy_files(self, source, destination, recursive=False):
             return result.exit_status == 0
         except process.CmdError as exc:
             raise NWException("failed to copy file {}".format(exc))
+
+    def remove_control_path(self):
+        """
+        Check for ssh connection if failed remove control path file
+        and reconnect it.
+
+        :returns: True if success or else raise exception
+        :rtype: bool
+        """
+        if not self.connect():
+            if not self.quit():
+                file_list = glob.glob("/root/.ssh/avocado-master-root*")
```

Comment:
You are assuming that the user is root. This is not going to be accepted. We need to make this dynamic. You could use `$HOME` env variable or even better: `os.path.expanduser()` method.

Reply:
ok,i will change it.